include/psa/crypto.h

```diff
@@ -124,14 +124,6 @@ psa_status_t psa_get_key_lifetime(psa_key_handle_t handle,
  * application calls psa_close_key() or psa_destroy_key() or until the
  * application terminates.
```

Comment:
How can we detect that the application has terminated and we can free the allocated key slot?

Reply:
That depends on the integration. For a library integration, when the application terminates, this frees all the runtime resources held by the crypto library. For a service integration, the access control layer in the service knows who each handle belongs to and should close a handle if it loses contact with the owner of that handle.